service/controller/chartconfig/v5/resource/chartstatus/create.go

```diff
@@ -34,6 +34,15 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 		customObjectCopy := customObject.DeepCopy()
 		customObjectCopy.Status.ReleaseStatus = releaseContent.Status
+
+		if releaseContent.Status != releaseStatusDeployed {
```

Comment:
I'm not into details but what if the status becomes `DEPLOYED` again?

Reply:
Good catch thanks. I think because of the deep copy we need to clear the status when the release is deployed. Otherwise incorrect information will be displayed.

@tomahawk28 WDYT?

Reply:
Thanks a lot, I just make a change of it.